bofire/data_models/kernels/continuous.py

```diff
@@ -20,6 +22,12 @@ class MaternKernel(ContinuousKernel):
     nu: float = 2.5
     lengthscale_prior: Optional[AnyPrior] = None
 
+    @validator("nu")
+    def validate_nu(cls, v, values):
```

Comment:
out of interest: what happened before the validator? Does BoTorch only support the `nu = p +1/2` cases?

Reply:
It was catched at the Gpytorch level and throws an error there if it is different from 0.5, 1.5 or 2.5. But we have not catched it in our data model, so it would fail at runtime and not before, which violates our concept ;)